assets/js/components/UI/Image.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
Can we change the title of this file to `assets/js/components/UI/WorkImage.jsx`?